css/components.css

```diff
@@ -5,15 +5,19 @@
 @import url('../component/Badges/badge.css');
 @import url('../component/Avatar/avatar.css');
 @import url('../component/Buttons/button.css');
+@import url('../css/style.css');
 
 :root{
-    --primary-color:#22c55e;
+    /* --primary-color:#22c55e; */
+    --primary-color:#16a34a;
     --secondary-color:#bbf7d0;
-    --primary-hover:#16a34a;
+    /* --primary-hover:#16a34a; */
+    --social-icon-color:#475569;
+    --primary-hover:#15803d;
```

Comment:
suggestion: just a suggestion you should give your color name as --color-primary.

Reply:
ok i will make these changes